infra/import_graph.py

```diff
@@ -0,0 +1,94 @@
+import networkx as nx
+
+def _type_prefix_match(prefixes, address):
+    """For a given address, find the type matching the address.
+
+    Takes an object containing an array of {prefix, type} pairs, and
+    an address. Returns the first type whose corresponding prefix
+    was a prefix of the given address."""
+    for prefix_type in prefixes:
+        prefix = prefix_type["prefix"]
+        if address[:len(prefix)] == prefix:
```

Comment:
```python
if address.startswith(prefix):
```

Reply:
:+1:

Reply:
This doesn't actually work--address is a list, not a string, and startswith is defined on strings.